tests/test_dle.py

```diff
@@ -270,33 +269,35 @@ class TestNodeParallelism(object):
         # and makes it sequential, as it doesn't like what happens in the inner dims,
         # where `x`, rather than `y`, is used
         (['Eq(t0, fc[x,x] + fd[x,y+1])', 'Eq(fc[x,x], t0 + 1)'],
-         (False, False)),
+         (False, True)),
         # outermost sequential w/ repeated dimensions
         (['Eq(t0, fc[x,x] + fd[x,y+1])', 'Eq(fc[x,x+1], t0 + 1)'],
-         (False, False)),
+         (False, True)),
         # outermost sequential, innermost sequential (classic skewing example)
         (['Eq(fc[x,y], fc[x,y+1] + fc[x-1,y])'],
          (False, False)),
         # outermost parallel, innermost sequential w/ double tensor write
         (['Eq(fc[x,y], fc[x,y+1] + fd[x-1,y])', 'Eq(fd[x-1,y+1], fd[x-1,y] + fc[x,y+1])'],
-         (True, False)),
+         (True, False, False)),
         # outermost sequential, innermost parallel w/ mixed dimensions
         (['Eq(fc[x+1,y], fc[x,y+1] + fc[x,y])', 'Eq(fc[x+1,y], 2. + fc[x,y+1])'],
          (False, True)),
     ])
-    def test_iterations_ompized(self, fa, fb, fc, fd, t0, t1, t2, t3,
-                                exprs, expected, iters):
-        scope = [fa, fb, fc, fd, t0, t1, t2, t3]
-        node_exprs = [Expression(DummyEq(EVAL(i, *scope))) for i in exprs]
-        iet = iters[6](iters[7](node_exprs))
-
-        parameters = derive_parameters(iet, True)
-        iet = Callable('kernel', iet, 'int', parameters)
-
-        iet = iet_analyze(iet)
+    def test_iterations_ompized(self, exprs, expected):
```

Comment:
Note: Ignore these changes, This is just to pass test, premptive pseudo merge of analysis PR will be ovewriteen